CMP1903_A1_2324/Die.cs

```diff
@@ -8,16 +8,29 @@ namespace CMP1903_A1_2324
 {
     internal class Die
     {
+
         /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
+        * The Die class should contain one property to hold the current die value,
+        * and one method that rolls the die, returns and integer and takes no parameters.
          */
 
-        //Property
-
-
-        //Method
+        Random random = new Random();
 
+        private int rollIt;
```

Comment:
'rollIt' could be confused as a function to make the die roll. Id recommend changing this to something to reflect the value of the rolled die. Maybe "outcome" or "number" would be more suitable.